### ISSUE TITLE
If an owner adds a new channel while a promo is active, generate code and send an email

### ISSUE THREAD

**Issue by ayumi** [brave-intl/publishers#488]
(Requires #466.)

When owners activate the Referral promo, they receive email confirmation with all their channel-specific referral codes/URLs (#469). 

While the promo is active, the owner might add new channels.
In this case, after verification we should:
- Generate a promo code via the Promo API (like in #466).
- Email the owner with the single channel-specific promo code/URL.

This task isn't required for the release of #466. However if we wait for {some time} after #466 is released, we may need to write a rake task to generate codes and send emails for relevant channels created during that {some time}. That can be covered in a separate issue.

(Thanks to @nvonpentz for the original thought)

**Comment by nvonpentz** [comment]
Implemented by #489
